src/Renderer.php

```diff
@@ -206,6 +209,91 @@ public function inlineJson($values)
         return htmlspecialchars(json_encode($values), ENT_QUOTES, 'UTF-8');
     }
 
+    /**
+     * Given a break stripe's content and index within the page, returns an array of list items for rendering in a table of contents
+     *
+     * @param  string $content the html in the textblock stripe as a string
+     * @param  int    $index   the index of the stripe within the page
+     * @return array
+     *
+     */
+    private function getBreakStripeTableItems($content, $index) {
```

Comment:
Avoid "table" in here since we'll have a table stripe, please. `getBreakStripeHeadings()`?

Reply:
(Same for textblock)